rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -130,6 +137,41 @@ private void doRecoverTransaction(Uid recoverUid) {
         }
     }
 
+    private void moveEntryToFailedLRAPath (final Uid failedUid) throws ObjectStoreException {
+        String failedLRAType = LongRunningAction.getFailedLraType();
+        InputObjectState inputState = _recoveryStore.read_committed(failedUid, _transactionType);
+        InputObjectState failedLRAUidState = _recoveryStore.read_committed(failedUid, failedLRAType);
+        boolean result = false;
+
+        if (inputState != null) {
+            if (failedLRAUidState != null) {
+                // Record already exists in failedLRARecord_PATH, hence removing it from the LRA_PATH
+                boolean rmRec = _recoveryStore.remove_committed(failedUid, _transactionType);
+                if (!rmRec) {
+                    LRALogger.logger.warnf("Unable to remove the failed duclicate failed LRA record (Uid: %s) " +
+                            "(which is already present in the failed LRA record location type: %s.) from LRA_PATH: %s",
```

Comment:
I would not use the word `LRA_PATH` as the type does not mean `PATH`. It's just location in the hierarchy, e.g. if you use JDBC object store then you do not work with a `PATH`.

Reply:
I didn't realize one thing (sorry for that). The convention is for the `error` and `warning` messages to use the `Logger` - ie. add the log under `lraI18NLogger`.

Reply:
ok